### PR TITLE
Favourite bh seeds

### PR DESCRIPTION
**Pull request by Superwhammybar**
Closes https://github.com/oldschoolgg/oldschoolbot/issues/5007

### Description:

Adding the ability to favourite seeds to use for birdhouse runs so that precious seeds, like herbs, can be avoided.

### Changes:

Added a new favorite_bh_seeds array on the user to store favourited seeds
Added `/config user favorite_bh_seeds` to allow for modification of favourites
Birdhouse activity uses the favourites list, if configured, and won't send a trip if there aren't any favourited seeds available

### Other checks:
- [x] I have tested all my changes thoroughly.

### CODE REVIEW

**Review by themrrobert — CHANGES_REQUESTED** [review]

File: src/lib/skilling/skills/hunter/birdHouseTrapping.ts [hunk 1]
```diff
@@ -178,4 +179,182 @@ const birdhouses: Birdhouse[] = [
...
+		amount: 10
+	},
+	{
+		name: 'Woad seed',
+		itemID: itemID('Woad seed'),
+		amount: 10
+	},
+	{
+		name: 'Limpwurt seed',
+		itemID: itemID('Limpwurt seed'),
```
Comment (gc) [inline src/lib/skilling/skills/hunter/birdHouseTrapping.ts:270]: Item name shouldn't be kept like this, manually written out (its hard to maintain and prone to mistakes). Instead, just do `item: getOSItem("Limpwurt seed")`. and then you get the item.name and item.id.

Comment (Superwhammybar) [reply src/lib/skilling/skills/hunter/birdHouseTrapping.ts]: Updated the code to use getOSItem and changed consumers to use `item.id`

File: src/lib/skilling/skills/hunter/birdHouseTrapping.ts [hunk 2]
```diff
@@ -178,4 +179,182 @@ const birdhouses: Birdhouse[] = [
 	}
 ];
 
+export const birdhouseSeeds = [
+	{
+		name: 'Hammerstone seed',
```
Comment (gc) [inline src/lib/skilling/skills/hunter/birdHouseTrapping.ts:184]: Im not sure if youre still working on it, but the names are still here.

Comment (Superwhammybar) [reply src/lib/skilling/skills/hunter/birdHouseTrapping.ts]: Sorry, completely missed this part! I've removed the variable and referenced `item.name` instead.

**Pull request merged**